### DISCUSSION TITLE
在 3.0.0.0-beta.3 中，在方法上使用了多个@Filter注解，接受到的消息将不会被过滤

### DISCUSSION THREAD

**Discussion by nosoul-nyl** [simple-robot/Discussions#39] [Q&A]
如标题所示

**Comment by nosoul-nyl** [comment]
具体使用的Filter 在下面代码中， 当然我使用@Filters  也是出了同样的问题

@Filter(value = "机器人",targets = @Filter.Targets(authors = "862197510"),matchType = MatchType.TEXT_EQUALS)
@Filter(value = "openai",targets = @Filter.Targets(authors = "862197510"),matchType = MatchType.TEXT_EQUALS)

**Reply by ForteScarlet** [reply]
首先先尝试更新到最新版本后再做尝试。如果仍有问题，请反馈到[issues](https://github.com/simple-robot/simpler-robot/issues)而并非 _讨论区_。
